packages/webcert/src/components/certificateList/CreateCertificateButton.tsx

```diff
@@ -1,19 +1,25 @@
 import { CustomButton, ResourceLink } from '@frontend/common'
 import file from '@frontend/common/src/images/file.svg'
 import React from 'react'
+import { useSelector } from 'react-redux'
+import { loadingCertificateTypes } from '../../store/patient/patientSelectors'
 
 interface Props extends ResourceLink {
   id: string
   onClick: (...args: string[]) => void
 }
 
-export const CreateCertificateButton: React.FC<Props> = ({ onClick, id, description, enabled, name }) => (
-  <CustomButton
-    buttonStyle="primary"
-    onClick={() => onClick(id)}
-    startIcon={<img src={file} alt={description} />}
-    disabled={!enabled}
-    text={name}
-    tooltip={description}
-  />
-)
+export const CreateCertificateButton: React.FC<Props> = ({ onClick, id, description, enabled, name }) => {
+  const isLoadingCertificateTypes = useSelector(loadingCertificateTypes)
```

Comment:
Instead of doing this from every button component, we can have a single `useSelector` in `CertificateList` and pass the disabled property to `CreateCertificateButton`